autosubmitconfigparser/config/configcommon.py

```
@@ -1185,13 +1210,73 @@ def check_conf_files(self, running_time=False, force_load=True, no_log=False):
+            return timedelta(hours=hours, minutes=minutes).total_seconds()
+
+        default_wallclock = _calculate_wallclock(self.experiment_data.get("CONFIG", {}).get("JOB_WALLCLOCK", "24:00"))
+        err_msg = ""
+        jobs = self.experiment_data.get("JOBS", {})
+        platforms = self.experiment_data.get("PLATFORMS", {})
+        wallclock_per_platform = {}
+
+        for platform_name in platforms.keys():
+            wallclock_per_platform[platform_name] = _calculate_wallclock(platforms[platform_name].get("MAX_WALLCLOCK",
+                                                                                                      self.experiment_data.get(
+                                                                                                          "CONFIG",
+                                                                                                          {}).get(
+                                                                                                          "JOB_WALLCLOCK",
+                                                                                                          "24:00")))
+
+        for job_name, job_data in jobs.items():
+            platform_wallclock = wallclock_per_platform.get(job_data.get("PLATFORM", ""), default_wallclock)
+            total_seconds = _calculate_wallclock(job_data.get("WALLCLOCK", "00:01"))
+            if total_seconds > platform_wallclock:
+                err_msg += f"Job {job_name} has a wallclock {total_seconds}s time greater than the platform's {total_seconds}s wallclock time\n"
+        return err_msg
+
+    def validate_jobs_conf(self) -> str:
+        """
+        Validate the job configurations.
+
+        :return: Error message if any validation fails, otherwise an empty string.
+        :rtype: str
+        """
+        err_msg = self.validate_wallclock()
+        return err_msg
+
+    def validate_config(self, running_time: bool) -> bool:
```

Out of curiosity; `running_time` appears to be `True` and raise `AutosubmitCritical` only for run, setstatus, and maybe monitor I think? But wouldn't it be better to always fail if the configuration is invalid?

---

For the create, it is because of the minimal->pre->conf->post. Maybe not all configurations are available. (Although, from memory I  think it is load twice one with running=False and other with running=True

For other commands, not all validation has to pass for it to do its work ( for example, the migrate only needs its stuff well configurated ), or maybe you want to use these commands to analyze what exactly is wrong ( for example, inspect, report ... ). Also at the start, we didn't have this experiment_data with all configuration load